packages/package_info/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.0
+
+* **Breaking change**. Change package_info to class PackageInfo.
```

Comment:
Should mention that this provides all package information with a single async call.